### PR TITLE
Intial commits

### PR DESCRIPTION
**Pull request by katakampranavi**
![sunbird-framework-management-tool-snapshot]- https://drive.google.com/file/d/1kdX56m_iqZAEbhI9EYB3T2ozhVmP4QQ7/view?usp=sharing

### CODE REVIEW

**Comment by HarishGangula** [comment]
The footer will be bottom of the page

**Comment by HarishGangula** [comment]
replace logo icon from. react to sunbird

File: sunbird-framework-management-tool/src/App.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        {/* Body Content */}
+        <Typography variant="h4" component="h1" align="center">
+          Welcome to My App!
+        </Typography>
+        <Typography variant="body1" component="p" align="center">
+          This is the body content of the app.
+        </Typography>
+      </Container>
+      <footer
+        style={{
```
Comment (HarishGangula) [inline sunbird-framework-management-tool/src/App.js]: No inline style allowed

**Review by HarishGangula — CHANGES_REQUESTED** [review]

**Comment by katakampranavi** [comment]
@HarishGangula As suggested i made the changes please find below link for snapshot
https://drive.google.com/file/d/1TD_1x0MdLSQ9L3baRBrTy-ywBbB-t6YK/view?usp=sharing

File: sunbird-framework-management-tool/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (HarishGangula) [inline sunbird-framework-management-tool/public/index.html]: Change this title to `sunbird framework management tool`

**Review by HarishGangula — CHANGES_REQUESTED** [review]

**Comment by katakampranavi** [comment]
@HarishGangula title name changed from react to sunbird framework management tool
https://drive.google.com/file/d/1XFUe1CGMlOE9ZFUMYbQ4kppI45CXUwK1/view?usp=sharing

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**